helm/values.yaml

```diff
@@ -13,15 +11,89 @@ sidecars:
   livenessProbeImage:
     repository: quay.io/k8scsi/livenessprobe
     tag: "v2.0.0"
+    pullPolicy: IfNotPresent
   nodeDriverRegistrarImage:
     repository: quay.io/k8scsi/csi-node-driver-registrar
     tag: "v1.3.0"
+    pullPolicy: IfNotPresent
 
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
+priorityClassName: system-node-critical
 
-podAnnotations: {}
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+rbac:
```

Comment:
this should be named less ambiguously, it means the psp-related rbac, not the other rbac resopurces  that are necessary for the driver to run